rholang/src/test/scala/coop/rchain/rholang/interpreter/ReplaySpec.scala

```diff
@@ -0,0 +1,131 @@
+package coop.rchain.rholang.interpreter
+
+import cats.effect.Resource
+import cats.syntax.all._
+import coop.rchain.crypto.hash.Blake2b512Random
+import coop.rchain.metrics
+import coop.rchain.metrics.{Metrics, NoopSpan, Span}
+import coop.rchain.rholang.Resources
+import coop.rchain.rholang.interpreter.accounting.utils.costLog
+import coop.rchain.rholang.interpreter.accounting.{_cost, Cost, CostAccounting}
+import coop.rchain.rspace.SoftCheckpoint
+import coop.rchain.shared.Log
+import monix.eval.Task
+import monix.execution.Scheduler.Implicits.global
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.concurrent.duration._
+
+class ReplaySpec extends FlatSpec with Matchers {
+
+  "multi joins (2/2)" should "execute successfully in replay" in {
+    val term =
```

Comment:
Have you tested situations with different types of sends and receives? Persistent/nonpersistent/peek?

Reply:
@Isaac-DeFrain I added comments for these tests to better explain the purpose. I see it as temporary just until all bugs with replay are resolved. This 500 iteration lasts about 10sec which is waste of time. We need better way to control execution of _Par_ in tests.